plotting-visualization/dicomtree.py

```diff
@@ -15,9 +15,12 @@
 
 from pydicom import compat
 
-if compat.in_py2:
-    import Tix as tkinter_tix
-else:
+try:
+    if compat.in_py2:
+        import Tix as tkinter_tix
+    else:
+        import tkinter.tix as tkinter_tix
+except:
```

Comment:
Better to use `except ImportError` to avoid catching other errors that might arise

Reply:
It should be instead `except AttributeError:`

to avoid this:
`AttributeError: module 'pydicom.compat' has no attribute 'in_py2'`

Reply:
Oh, that is because pydicom no longer supports Python 2, so the `in_py2` flag has been removed.  The `contrib` scripts should by now remove all references to `in_py2` and assume pydicom >= 2.0.